### ISSUE TITLE
Status code 0 is not a good one

### ISSUE THREAD

**Issue by gagoman** [zalando-stups/yourturn#514]
`err.status || 0` would fail an entire app if no `err.status` would be found.

If no code is available it is better to return `500` or something like that.

**Comment by prayerslayer** [comment]
True, not sure how this got in there. Hot needle and copy-paste, probably.
